Math/Binary_Exponentiation/binaryExponentiation.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
```

Comment:
do you think a comment about what the code is doing will be better along with time/space complexity?